Update GOVERNANCE.md (#313)
numfocus/DISCOVER-Cookbook: GOVERNANCE.md

  - Continuous Integration: Using Github Actions we are able to deploy the guide, preview pull-requests, and run quality checks on content.

# **Roles and Definitions**
* Contributors
    - *Description:*
        - Any person who makes a contribution to the DISCOVER Cookbook
    - *Responsibilities:*
        - Submit issues, ideas, pull requests
        - maintain the code of conduct

>         - reveiw pull requests

Reviewer: ```suggestion
        - review pull requests
```

    - *Eligibility:*

>         - no criteria on people to accept, 

Reviewer: I don't know what this means

Author: Hmm was trying to communicate that anyone can submit an issue, pull request, discussion items. I think we can just delete this line and change the next line to:

```
- must have an accepted issue, pull request, or idea in discussions (either on discussion boards or at in person event
```

Reviewer 2: - *Eligibility:*
  - Anyone can submit issues, pull requests, or discussion items to the project
  - To be recognized as a Contributor, one must have at least one accepted contribution (issue, pull request, or discussion item) in the project.

        - once a contribution is accepted, will be add to the contributor pages
    - *Current Team:*
        - see [All Contributors File](https://github.com/numfocus/DISCOVER-Cookbook/blob/main/.all-contributorsrc)

* Maintainer
    - *Description:*
        - Experienced project members who have made numerous contributions and is willing to take the responsibility for maintaining the repository
    - *Responsibilities:*
        - triage issues,
        - maintain content delivery,
        - Write control: Approve pull/merge requests (verify submissions vs. acceptance criteria)
        - Monitor github issues and drive timely resolution
        - Ensure project information such as the Readme, Contributing docs, wiki, etc., kept up-to-date
        - Lead the project in terms of versioning, quality assurance, and overarching objectives
    - *Eligibility:*
        - Needs to be familiar with GitHub-run projects and lifecycles of open source projects

>         - Needs to have contributed to the project significantly so that other maintainers can trust their decision

Reviewer: ```suggestion
        - Needs to have contributed to the project significantly so that other maintainers can trust their decisions
```

        - Needs to have time each week to respond to folks
    - *Current Team:*
        - see [DISCOVER COOKBOOK Maintainers Team](https://github.com/orgs/numfocus/teams/discover-cookbook-maintainers)
* Editor:
    - *Description:*
        - Responsibile for ensuring content is reviewed and kept to a high standard. 
    - *Responsibilities:*
        - maintain content
        - review content issue
    - *Eligibility:*
        - Invitation only based on roles in NumFOCUS DISC 
        - Must be able to attend editorial meetings roughly once per month
    - *Current Team:*
        - see [DISCOVER COOKBOOK Editors](https://github.com/orgs/numfocus/teams/discover-cookbook-editors)
* NumFOCUS DISC Committee
    - *Description:*

>         - members of the disc committee as elected per their charter 

Reviewer: ```suggestion
        - members of the DISC committee as elected per its charter 
```

    - Approval is dependent upon adherence to the guidelines in [CONTRIBUTING.md](CONTRIBUTING.md), and alignment with
repository goals of maintainability, completeness, and quality
    - Conflicts or questions will ultimately be resolved by the NumFOCUS DISC Committee

# Processes

- Internal system changes are handled by maintainers directly, without others needing to be involved

> - How are new contributions managed

Reviewer: ```suggestion
- How new contributions are managed
```

    - A github pull request is made by anyone, preferred to be referencing a github issue unless very small change
    - A maintainer triages the issue.
        - content is tagged and assign to the editorial team to review
        - other tags are handled by maintainers
    - Revewers for a github pull request work with the contributor to approve the pull request
    - Once a pull request is approved, a maintainer will merge to the repository.

>     - Once merged maintaners will add those involved to the contributors list via all-contributors

Reviewer: ```suggestion
    - Once merged maintainers will add those involved to the contributors list via all-contributors
```

> - How do we review existing content

Reviewer: ```suggestion
- How existing content is reviewed
```

    - types of edits
        - copy edits
        - rewrite for clarity
        - changing content
    - let anyone make issue or pull requests

>     - monthly meeting we can review in person, but could be done adhoc

Reviewer: ```suggestion
    - monthly meeting we can review in person, but could be done ad hoc
```

    - At the monthly meeting will review
        1. open issues tagged as content related
        2. review preassigned chapters

> - How do we determine which new content to add

Reviewer: ```suggestion
- How we determine which new content to add
```

    - Someone proposes new content either in a editorial meeting or on github discussions / issues
    - Editorial team votes on adding the chapter
    - Editorial team opens an issue saying that new content in an area is being developed

>     - An editor writes or composes the chapter from contributions

Reviewer: ```suggestion
    - An editor writes or composes new content from contributions
```

>       - Alternative, we see if anyone external wants to make contributions or take the lead in drafting a new chapter

Reviewer: ```suggestion
      - Alternative, we see if anyone external wants to make contributions or take the lead in drafting the new content
```

>     - The new content uses the reviewing proces above

Reviewer: ```suggestion
    - We use the reviewing process above for the new content 
```
